booking-engine-api/operations/services.md

```diff
@@ -0,0 +1,354 @@
+                    "NetValue": 9090.91,
+                    "TaxValue": 909.09
+                  },
+                  {
+                    "TaxRateCode": "CZ-Z",
+                    "NetValue": 1.02,
+                    "TaxValue": 0.00
+                  }
+                ]
+              }
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RateGroups` | array of [Rate group](hotels.md#rate-group) | required | Information about all available rate groups. |
+| `Rates` | array of [Rate](hotels.md#rate) | required | Information about all available rates. |
+| `CategoryPrices` | array of [Category price](#category-price) | required | Prices for all specified categories. |
+
+#### Category price 
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | required | Unique identifier of category. |
+| `OccupancyPrices` | array of objects with arrays of [Occupancy](#occupancy) | required | List of occupancies for age categories against which rate group prices are supplied. |
+| `RateGroupPrices` | array of [Rate group price](#rate-group-price) | required | Prices for the given category for each of the occupancy bands specified in OccupancyPrices. |
+
+#### Occupancy
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AgeCategoryId` | string | required | Identifier of age category. |
+| `PersonCount` | number | required | Number of persons of this Age category type. |
+
+#### Rate group price
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `MinRateId` | string | required | Unique identifier of rate. |
+| `MinPrice` | [Amount](#amount) objects | required | TotalAmount contains total price for the specified interval for rate, AverageAmountPerTimeUnit contains price for one timeUnit. (Day/Night) |
+| `MaxPrice` | [Amount](#amount) objects | required | TotalAmount contains total price for the specified interval for rate, AverageAmountPerTimeUnit contains price for one timeUnit. (Day/Night) |
+
+#### Rate
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the rate. |
+| `Name` | [Localized text](hotels.md#localized-text) | required | Name of the rate localized into all supported languages. |
+| `Description` | [Localized text](hotels.md#localized-text) | required | Description of the rate localized into all supported languages. |
+| `IsPrivate` | boolean | required | Set to `true` for promotion rate enabled by provided `VoucherCode`. |
+| `ServiceId` | string | required | Unique identifier of the [Service](configuration.md#service) to which the rate is bound. |
+| `RateGroupId` | string | required | [Rate group](hotels.md#rate-group) Identifier of the rate. |
+| `Ordering` | number | required | Number defining the ordering of the rate. |
+
+
+#### Amount
```

Comment:
`Amount` should be defined, not just an example of data. If it's defined somewhere else in the API, you could point to that? Otherwise, it needs to be defined here.

Reply:
added definition